conf/messages.conf

```diff
@@ -289,7 +289,8 @@
 287: You cannot change party leaders in this map.
 //Missing stuff for @killer related commands.
 288: You are no longer killable.
-//289-290 FREE
+289: Target character must be in the same map.
+//290 FREE
```

Comment:
I suggest to use msgstring for 20120410 clients onwards and use `message.conf` on lower client. And also please use the same message text on msgstringtable.txt